alvr/egui_dashboard/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "alvr_egui_dashboard"
+version = "15.2.1"
+authors = ["alvr-org"]
+license = "MIT"
+edition = "2018"
+
+[dependencies]
+eframe = "0.13"
+alvr_common = { path = "../common" }
```

Comment:
Missing trailing newline

Reply:
You are quite picky :) This crate will be merged into alvr_server once the dashboard is finished and will replace the current dashboard.